apps/blog/src/posts/transitions-demo/transitions-demo.md

```diff
@@ -0,0 +1,172 @@
+
+## Container Transform 
+
+Thus far, we have focused on transitioning between pages with unrelated content. What if there were elements on one page that we wanted to persist to the next? That is where the Container Transform transition type should be used. This transition type has a lot of moving pieces, so it is important to remember to keep the transition clean. We do not need any extra bouncing movements as the transition completes, we want to move our elements along one axis and make sure the elements do not overlap or cause other elements to shift. You can see examples of what NOT to do for this transition type on the Material Designs demonstration.  
+
+In my sample app, my “persistent element” was an album art and when clicked, I wanted it to come front and center on a new page with the respective track list. Like using startViewTransition page to page, using it for an element automatically does a default sliding animation from its old location to its new location. The Typescript is very close to that of the forward and backward transition: 
+
+```js
+async handleAlbumClick(name: string, index: number){ 
+  let album: HTMLElement = this.shadowRoot!.querySelector(`[data-name="${name}"]`)! 
+  if ("startViewTransition" in document) { 
+    //@ts-ignore 
+    album!.style.viewTransitionName = 'container-transform'; 
+    // @ts-ignore 
+    const transition = document.startViewTransition(); 
+    Router.go(`/album-details/${name}/${index}`); 
+    await transition.finished; 
+    // @ts-ignore 
+    album!.style.viewTransitionName = ''; 
+  }
+```
+
+And to link the album cover on the old page to the album on the new page, I gave the view-transition-name of “container-transform” to the album art on the new page. Additionally, I wanted the track list to reveal itself as a part of the animation. Since the track list has nothing to do with page transitions, I was able to use normal CSS animations to achieve the reveal I was looking for. A reminder that using the View Transitions API in tandem with the CSS animations that we are used to is completely acceptable.  
+
+
+We have only just begun to explore the power of page transitions and with this technology only being a few months old, there is only more to come. If you want to do some reading about the feature I would recommend the Chrome Developer article which breaks down the API completely with code examples via a demo app. For more information about the different types of transitions and when to use them, check out the Material Design article that I mentioned a few times through the course of this blog post. I am excited to see this API be used more widespread and who knows, maybe you’ll see some of this on the PWABuilder site soon!  
```

Comment:
Can you add the link to this chrome developer article? Or did I miss it somewhere in the previous content?